dotnet-interactive/HttpApiBootstrapper.cs

```diff
@@ -10,36 +10,38 @@ internal static class HttpApiBootstrapper
     {
         public static string GetJSCode(Uri apiRoot)
         {
-            var id = $"{Process.GetCurrentProcess().Id}.{apiRoot.Port}";
             var template = @"#!javascript
-if ((typeof(requirejs) !==  typeof(Function)) || (typeof(requirejs.config) !== typeof(Function))) { 
-    let script = document.createElement(""script""); 
-    script.setAttribute(""src"", ""https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js""); 
+if ((typeof(require) !==  typeof(Function)) || (typeof(require.config) !== typeof(Function))) {
+    let script = document.createElement('script');
+    script.setAttribute('src', 'https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js');
     script.onload = function(){
         loadDotnetInteractiveApi();
     };
-    document.getElementsByTagName(""head"")[0].appendChild(script); 
+    document.getElementsByTagName('head')[0].appendChild(script);
 }
 else {
     loadDotnetInteractiveApi();
 }
 
 function loadDotnetInteractiveApi(){
-    let apiRequire = requirejs.config({context:""dotnet-interactive.$SEED$"",paths:{dotnetInteractive:""$API_URL$""}});
```

Comment:
restarting kernel will change port, this seed into the name is to prevent cache and avoid restarting the browser tag completely and loosing the work.